packages/@okta/vuepress-site/docs/reference/rest/index.md

```diff
@@ -0,0 +1,360 @@
+
+To import a collection from the Okta [Postman Collections](https://developer.okta.com/docs/reference/postman-collections/) reference page:
+
+1. Click **Run in Postman** next to the collection you want.
+
+   For example, click this **Run in Postman** action to import the Users API collection:
+   [![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/9daeb4b935a423c39009)
+   > **Note:** The **Run in Postman** option is also available on each [core API reference](/docs/reference/core-okta-api/) page on this site.
+
+1. On the **Run in** dialog, either select to import your collection to your local Postman app or to your web Postman account.
+
+   If you select the web option, you need to sign in to your online Postman account.
+
+1. Select the workspace to import your collection, and then click **Import**.
+
+## Set up Okta for API access
+
+To access Okta APIs from Postman, you need to authenticate with the Okta API resource server. Okta APIs support the OAuth 2.0 authentication scheme that uses access tokens. Access tokens enable the bearer to perform specific actions on specific Okta endpoints, defined by the scopes in the token.
+
+Scoped access tokens have several advantages, including:
+
+* More access granularity
+* Shorter token lifespans
+* Can be generated and retrieved using an API
+
+> **Note:** Okta doesn't recommend using the Okta-propriety `SSWS` API token authentication scheme.
+> This API token scheme allows you to access a broad range of APIs since there's no scope associated with the token. Access to the APIs depends on the privileges of the user that [created the API token](/docs/guides/create-an-api-token/main/). The API token also has a fixed expiry date.
+
+You need to obtain an OAuth 2.0 access token to configure the authorization header of your Postman API requests to Okta. To obtain this access token, set up Okta for your use case:
+
+* [User-based API access setup](#user-based-api-access-setup): to obtain an access token that is scoped for specific resources and actions and tied to the permissions of an Okta user
+* [Service-based API access setup](#service-based-api-access-setup): to obtain an access token that is scoped for specific resources and actions, not associated with an Okta user
+
+### User-based API access setup
+
+If your use case requires you to access a limited number of Okta endpoints as a specific user, use the OAuth 2.0 Authorization Code grant flow with PKCE. See the following task to set up your Okta org for API access.
+
+#### Create an OIDC app in Okta
+
+Create an OIDC app integration to define your scope-based access to Okta APIs with the Authorization Code grant flow with PKCE.
+
+1. [Sign in](https://developer.okta.com/login) to your Okta org Admin Console as a user with administrative privileges (Super Admin role).
```

Comment:
```suggestion
1. [Sign in](https://developer.okta.com/login) to your Admin Console as a user with administrative privileges (super admin role).
```

Reply:
"Okta org" doesn't seem necessary to me, but I'll defer to you here. Nothing wrong with it stylistically. We do lowercase the admin roles, though, unless we're referring to labels the capitalize them